### ISSUE TITLE
Make user interface updates trigger notifications to open notifications

### ISSUE THREAD

**Issue by ernout-worth** [open-zaak/open-zaak#1759]
### Thema / Theme

Zaken API

### Omschrijving / Description

It seems notifications to open notifications are only sent if you use open-zaak's API endpoint.
But is that how we want it to work?

With objecttypen changes it will also post updates you do through the user interface.
But when I create a new zaak through the user interface, it won't notify open notificaties, only when I use the API.

### Toegevoegde waarde / Added value

This way we can rely on being informed on relevant updates of objects, wether a change is made through the user interface or through the API is not relevant imo.

**Comment by joeribekker** [comment]
@ernout-worth I agree but this can be tackled in 2 ways: A "proper" UI that works on top of the API will tackle this; or we hook the current admin UI into notifications that is duplicated behaviour from API events. We do the latter already in some parts but not everywhere.

We need a stakeholder for this.

Estimate for a new UI is out of scope, but changing the admin (personally, I think the the new UI is preferred): This might go up to 5 weeks of work.

**Comment by joeribekker** [comment]
Refinement: This will be tackled by LBI (Lekkere Beheer Interface) and I dont think we will get any budget from others for this. Closing but if budget arrives, we could re-open it.
